goal_src/engine/sparticle/sparticle.gc

```diff
@@ -677,45 +680,7 @@
   )
 
 (defun set-particle-frame-time ((arg0 int))
-  (cond
-   ((= arg0 5)
-    (set-vector!
-     *sp-frame-time*
-     0.00000000000000000000000000000000000001175495
-     5.0
-     1.0
-     1.0
-     )
-    )
-   ((= arg0 6)
-    (set-vector!
-     *sp-frame-time*
-     0.000000000000000000000000000000000000011754952
-     6.0
-     1.2
-     1.2
-     )
-    )
-   ((= arg0 10)
-    (set-vector!
-     *sp-frame-time*
-     0.000000000000000000000000000000000000011754958
-     10.0
-     2.0
-     2.0
-     )
-    )
-   ((= arg0 12)
-    (set-vector!
-     *sp-frame-time*
-     0.00000000000000000000000000000000000001175496
-     12.0
-     2.4
-     2.4
-     )
-    )
-   )
-  0
+  (set-vector! *sp-frame-time* (the-as float (logior #x800000 arg0)) (the float arg0) (* 0.2 arg0) (* 0.2 arg0))
```

Comment:
Might be worth a comment saying that this was rewritten.

I don't think it matters, but I think this will have slightly different behavior if the game is running at 15 fps.